packages/cli/templates/default-plugin/src/components/ExampleComponent/ExampleComponent.test.tsx.hbs

```diff
@@ -17,10 +17,15 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import ExampleComponent from './ExampleComponent';
+import { ThemeProvider } from '@material-ui/core';
+import { BackstageTheme } from '@spotify-backstage/core';
 
 describe('ExampleComponent', () => {
   it('should render', () => {
-    const rendered = render(<ExampleComponent />);
+    const rendered = render(
+      <ThemeProvider theme={BackstageTheme}>
+        <ExampleComponent />
+      </ThemeProvider>,);
```

Comment:
Does this lint?

Reply:
I'll remove the comma

Reply:
If you set vscode to tax (in the lower right corner) it'll lint/prettier it for you

Reply:
Oh, wait, prettier is adding the comma. Also, regarding linting, I think there's a bunch of things that isn't linting right now, we need to go through those warnings and clean up.